src/EditorFeatures/TestUtilities/RefactoringHelpers/RefactoringHelpersTestBase.cs

```diff
@@ -20,20 +19,18 @@
 namespace Microsoft.CodeAnalysis.Editor.UnitTests.RefactoringHelpers
 {
     [UseExportProvider]
-    public abstract class RefactoringHelpersTestBase<TWorkspaceFixture> : TestBase, IClassFixture<TWorkspaceFixture>, IDisposable
+    public abstract class RefactoringHelpersTestBase<TWorkspaceFixture> : TestBase
         where TWorkspaceFixture : TestWorkspaceFixture, new()
     {
-        protected readonly TWorkspaceFixture fixture;
+        private readonly TestFixtureHelper<TWorkspaceFixture> _fixtureHelper;
 
-        protected RefactoringHelpersTestBase(TWorkspaceFixture workspaceFixture)
-            => this.fixture = workspaceFixture;
-
-        public override void Dispose()
+        protected RefactoringHelpersTestBase()
         {
-            this.fixture.DisposeAfterTest();
```

Comment:
📝 The `IDisposable.Dispose` method runs _after_ the `UseExportProviderAttribute.After` method, which means it is not capable of disposing a workspace instance used by a test. All impacted tests that I found have been updated to dispose of the workspace in the correct order.